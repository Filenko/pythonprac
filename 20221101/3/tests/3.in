f, g, h = Grange(1, 2, 1048575), Grange(1, 2, 1048576), Grange(1, 2, 1048577)
o, e = Grange(1, 2, 1), Grange(100, 10, 200)
print(f"{f=}, {g=}, {h=}")
print(f"{list(o)}({not(o)}), {list(e)}({not(e)})")
print(f"{len(f)=}, {len(g)=}, {len(h)=}")
print(f"{len(list(f))=}, {list(g)=}, {len(list(h))=}")
print(f"{f[0]=}, {h[24]=}")
a, b = f[3:3276], h[3:32767:34]
print(f"{a=}, {a[5]=}")
print(f"{b=}, {b[0]=}, {list(b)=}")
