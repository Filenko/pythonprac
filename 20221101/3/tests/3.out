f=grange(1, 2, 1048575), g=grange(1, 2, 1048576), h=grange(1, 2, 1048577)
[](True), [100](False)
len(f)=20, len(g)=20, len(h)=21
len(list(f))=20, list(g)=[1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536, 131072, 262144, 524288], len(list(h))=21
f[0]=1, h[24]=16777216
a=grange(3, 2, 3276), a[5]=96
b=grange(3, 17179869184, 32767), b[0]=3, list(b)=[3]
